uberfire-extensions/uberfire-wires/uberfire-wires-core/uberfire-wires-core-grids/src/main/java/org/uberfire/ext/wires/core/grids/client/widget/grid/impl/BaseGridWidget.java

```diff
@@ -244,6 +248,12 @@ public double getHeight() {
         return height;
```

Comment:
Should `getHeight()` now be deprecated?

Reply:
No, a grid still has height, as do rows and now cells. A row's height is that of it's tallest cell and a grids height is the sum of its rows heights.